stablehlo/dialect/StablehloOps.td

```diff
@@ -1708,19 +1709,19 @@ def StableHLO_BatchNormGradOp : StableHLO_Op<"batch_norm_grad", [Pure,
   }];
 
   let arguments = (ins
-    RankedTensorOf<[HLO_Float]>:$operand, /*batch_norm_grad_i1*/
-    1DTensorOf<[HLO_Float]>:$scale, /*batch_norm_grad_i2*/
-    1DTensorOf<[HLO_Float]>:$mean, /*batch_norm_grad_i3*/
-    1DTensorOf<[HLO_Float]>:$variance, /*batch_norm_grad_i4*/
-    RankedTensorOf<[HLO_Float]>:$grad_output, /*batch_norm_grad_i5*/
+    RankedTensorOf<[HLO_Float, HLO_QuantizedInt]>:$operand, /*batch_norm_grad_i1*/
```

Comment:
Left on a future CL, but let's add a ticket to make this `FpTensor` once there is no unranked tensor type in StableHLO

Reply:
for now, prefer not to change it to FpTensor.  Created ticket to review this once all Q verifier changes are done.

Reply:
revisited: there are 6 ops which doesn't take quantized types, few also take other types like complex, it appear ok to keep float and quant separate and use HLO_FpOrQuantizedIntTensor .